src/streamlink_cli/argparser.py

```diff
@@ -1449,6 +1461,7 @@ def build_parser():
     ("ffmpeg_no_validation", "ffmpeg-no-validation", None),
     ("ffmpeg_verbose", "ffmpeg-verbose", None),
     ("ffmpeg_verbose_path", "ffmpeg-verbose-path", None),
+    ("ffmpeg-loglevel", "ffmpeg-loglevel", None),
```

Comment:
Why did GitHub eat my review comment here? I shouldn't have refreshed the page after the second commit was pushed...

----

The first value of this tuple is the `argparse.Namespace` attribute that'll be set. This can't be in a dash notation, or it'll raise a

> ```
> AttributeError: 'Namespace' object has no attribute 'ffmpeg-loglevel'. Did you mean: 'ffmpeg_loglevel'?
> ```